cirq/google/ops/physical_z_tag.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A class that can be used to denote a physical Z gate."""
+from cirq import protocols
+
+
+class PhysicalZTag:
+    """Class to add as a tag onto an Operation to denote a Physical Z operation.
+
+  By default, all Z rotations on Google devices are considered to be virtual.
+  When performing the Z operation, the device will update its internal phase
+  tracking mechanisms, essentially commuting it forwards through the circuit
+  until it hits a non-commuting operation (Such as a sqrt(iSwap)).
+
+  By attaching this to a Z rotation operation, this class will denote to the
```

Comment:
nit: "When applied to a Z rotation operation, this tag indicates to the hardware that..."

Reply:
Done.